CHANGELOG.md

```diff
@@ -28,7 +31,7 @@ _None_
 
 * Improved installation instructions in the README.  
   [Olivier Halligon](https://github.com/alisoftware)
-  [#303](https://github.com/SwiftGen/SwiftGenKit/issues/303)
+  [#303](https://github.com/SwiftGen/SwiftGen/issues/303)
```

Comment:
We can at least check if it's a broken link, and that the issue number matches the link's number (we've made that mistake too).